tester_app/client/assets/stylesheets/_bootstrap-variables-customization.scss

```diff
@@ -0,0 +1,4 @@
+// This file is used as part of the bootstrap-sass-loader customization
+// Included from bootstrap-sass.config.js
+// Use it to apply styles that need to go BEFORE bootstrap has loaded
```

Comment:
// Use it to define variables BEFORE bootstrap loads. See http://getbootstrap.com/customize/